client/ayon_zbrush/hooks/pre_subprocess_no_window.py

```diff
@@ -0,0 +1,21 @@
+import subprocess
+from ayon_applications import PreLaunchHook, LaunchTypes
+
+
+class LaunchServerNoWindow(PreLaunchHook):
+    """Specifically for Zbrush to make the AYON tools launching faster
+    """
+
+    # Should be as last hook because must change launch arguments to string
+    order = 1000
+    app_groups = {"zbrush"}
+    platforms = {"windows"}
+    launch_types = {LaunchTypes.local}
+
+    def execute(self):
+
+        self.launch_context.kwargs.update({
```

Comment:
I think this will make ZBrush process as child of AYON launcher process.

BTW Can you this change with AYON launcher build when using `ayon.exe`?